lib/utils.py

```diff
@@ -173,10 +173,11 @@ def warn(message):
         message(str): The message which will be showed to the user.
     
     Returns:
-        returns None if the answer is yes
+        returns True if user confirms with yes
     """
-
     resp = input(message + " [y/n]")
```

Comment:
Sorry, missed this the first time. Capitalize the N to indicate no is the default response.